akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -744,7 +744,7 @@ private[remote] class EndpointManager(conf: Config, log: LoggingAdapter)
           endpoint ! s
         case Some(Gated(timeOfRelease)) =>
           if (timeOfRelease.isOverdue()) createAndRegisterWritingEndpoint() ! s
-          else extendedSystem.deadLetters ! s
+          else extendedSystem.deadLetters                                   ! s
```

Comment:
this alignment seems odd - is this on purpose?

Reply:
Yes, its a change which aligns the `!` operator which is used in actors.

Reply:
I don't think this is an improvement. In such a case it would probably be better to manually reformat the if/else to separate lines to keep things more in line

Reply:
Yeah now that I see the effect I agree, will change it later. The only other operator that comes to mind which makes sense to align this way is `->` for setting values in `Map`'s

Reply:
I have just force pushed the PR with the `!` and `?` alignment operators removed.